task_scheduler/configs/config.py

```diff
@@ -0,0 +1,31 @@
+# there should be a configuration class that can return data about:
+#- the db port of my db for my application
+#- the ip/hostname of my db db for my application
+#- the log level for my custom logger (this is for the Customlogger to consume)
+import os
+import json
+from json import JSONDecodeError
+
+class Configuration:
+    '''Loads the configuration json file (depending on FLASK_ENV variable)
+     and returns a dict. If loading fails prints error and returns enmpty dict
+    '''
+    def __init__(self):
+        self.configuration = {}
+        filename = ''
+        if os.environ.get('FLASK_ENV') == 'development':
+            filename = 'task_scheduler/configs/config.dev.json'
```

Comment:
maybe here you can use `path` to get the absolute path, somthign like the following
`absulte_path= path.join(path.abspath(path.dirname(__file__)), 'config.dev.json') `

Reply:
Thanks the suggestions have been applied.